lenny/routes/api.py

```diff
@@ -135,26 +134,29 @@ async def proxy_readium(request: Request, book_id: str, readium_path: str, forma
     content_type = r.headers.get("Content-Type", "application/octet-stream")
     return Response(content=r.content, media_type=content_type)
 
-@router.post('/items/{book_id}/borrow', status_code=status.HTTP_200_OK)
+@router.get('/items/{book_id}/borrow')
```

Comment:
This risks sending sensitive information over GET -- this should be on a separate endpoint